### PR TITLE
Release prep/v0.59.0

### PR DESCRIPTION
**Pull request by Aneurysm9**

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
Look like the latest merged PR is missed https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13743

@Aneurysm9 do you want to submit another follow-up PR or update this one?

**Comment by Aneurysm9** [comment]
> Look like the latest merged PR is missed #13743
> 
> @Aneurysm9 do you want to submit another follow-up PR or update this one?

I can update this one, but we need to freeze `main` or it'll always be behind :)

**Comment by dmitryax** [comment]
Sounds good.

@open-telemetry/collector-contrib-maintainer let's avoid merging other PRs until this one is merged :)

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**
